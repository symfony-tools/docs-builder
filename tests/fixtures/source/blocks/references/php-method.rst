
:phpmethod:`Locale::getDefault`

:phpmethod:`BcMath\\Number::add`
