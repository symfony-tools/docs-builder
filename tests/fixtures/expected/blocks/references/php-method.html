<p><a href="https://secure.php.net/manual/en/locale.getdefault.php" class="reference external" title="Locale::getDefault()" rel="external noopener noreferrer" target="_blank">getDefault()</a></p>
<p><a href="https://secure.php.net/manual/en/bcmath-number.add.php" class="reference external" title="BcMath\Number::add()" rel="external noopener noreferrer" target="_blank">add()</a></p>
